src/logger/Logger.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+import os
+from enum import Enum
+
+class Severity(Enum):
+    DEBUG = 1
+    INFO = 2
+    WARNING = 3
+    ERROR = 4
+
+class Logger:
+    
+    def __init__(self, debug=False, filename='log.txt'):
+        self.debug = debug
+        self.filename = filename
+        self._f = open(self.filename,'a+')
+
+        self._f.write('[{}]  {}  {} \n'.format( 'INFO', str(datetime.datetime.now()), 'Logger started.'))
+
+
+    def log(self, message, severity=Severity.INFO):
+        timestamp = datetime.datetime.now()
+
+        if severity == Severity.ERROR:
+            self._f.write('[{}]  {}  {} \n'.format('ERROR', str(timestamp), message))
+        elif severity == Severity.INFO:
+            self._f.write('[{}]  {}  {} \n'.format('INFO', str(timestamp), message))
+        elif severity == Severity.WARNING:
+            self._f.write('[{}]  {}  {} \n'.format('WARNING', str(timestamp), message))
+        elif severity == Severity.DEBUG:
+            if self.debug:
+                self._f.write('[{}]  {}  {} \n'.format('DEBUG', str(timestamp), message))
+                
+        self._f.flush()
+        os.fsync(self._f)
+
+    def shutdown(self):
```

Comment:
I would argue this method should be named `__del__`, Python's deconstructor, so we don't need to worry about calling shutdown. Additionally, if the timing is important, we could do `del logger_instance` to delete it right then and there.

Reply:
I'm going to leave it the way it is for this sprint but you make a good point that the cleanup actions for the constructor should probably be in the destructor.

Reply:
I think we should fix it now since it's a super easy change and any code next sprint that uses this class will have to be changed, or wait until this is changed before using this class.